### DISCUSSION TITLE
Bug "-new-templates" not work

### DISCUSSION THREAD

**Discussion by ipsecc** [projectdiscovery/nuclei#4517] [Q&A] [answered]
## Description
Parameter "-new-templates" does not work, the number of requests is always 7432.

## Steps To Reproduce:
`nuclei -u https://www.hackerone.com/ -s medium,high,critical -stats`
![image](https://github.com/projectdiscovery/nuclei/assets/40918024/3f482170-0345-4b62-bd5e-0ae0b62b8300)

**Add "-nt"**

`nuclei -u https://www.hackerone.com/ -s medium,high,critical -nt -stats`
![image](https://github.com/projectdiscovery/nuclei/assets/40918024/fa39315d-c8bd-4f5a-821c-7769186834df)

**Comment by ehsandeep** [comment, marked as answer]
@ipsecc this is is not a bug, there is no new templates added in last template project release - https://github.com/projectdiscovery/nuclei-templates/releases/tag/v9.7.1

if you wanted to run newly added templates from specifc version, you can do it using `-ntv v9.7.0` for example.

**Comment by ipsecc** [comment]
@ehsandeep Thanks for the reply. If there are no new templates, I think the scan should be aborted instead of scan all the templates. What do you think?
